src/apps/utils/languageHelpers.js

```diff
@@ -0,0 +1,11 @@
+export function isCustomLanguage(name) {
+  return !Object.prototype.hasOwnProperty.call(CONFIG.A5E.languages, name);
+}
+
+export function localizeLanguage(name) {
+  if (isCustomLanguage(name)) {
+    return name;
+  }
+
+  return game.i18n.localize(CONFIG.A5E.languages[name]);
+}
```

Comment:
We keep our helper functions to 1 per file. Could you move this to a separate file and use default exports for all of these language utility functions.

Reply:
By the way, it seems to me that we can simplify this quite a bit and completely do away with the `isCustomLanguage` check.

I believe this provides identical functionality:
```js
export function localizeLanguage(name) {
  return game.i18n.localize(CONFIG.A5E.languages[name] ?? name);
}
```

Reply:
Also, same comment as before regarding the use of `name` as the parameter here instead of something more specific like `language`. We should probably go for this:
```js
export function localizeLanguage(language) {
  return game.i18n.localize(CONFIG.A5E.languages[language] ?? language);
}
```